apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -340,6 +340,17 @@ type ClusterResourcePlacementStatus struct {
 	// +optional
 	SelectedResources []ResourceIdentifier `json:"selectedResources,omitempty"`
 
+	// ObservedResourceIndex is the resource index of a group of ClusterResourceSnapshot CR that the conditions in the
```

Comment:
Explain resource index first? Something like this?

Resource index logically represents the generation of the selected resources. We take a new snapshot of the selected resources whenever the selection or their content change. Each snapshot has a different resource index.

ObservedResourceIndex is the resource index that the conditions in the ClusterResourcePlacementStatus observe.

To get clusterResourceSnapshot of a given resource index, use the following command:

""